blox.cue

```diff
@@ -1,5 +1,4 @@
 {
-    data_dir: "dogfood/data/source"
-    schema_dir: "dogfood/data/schema"
-    build_dir: "_build"
+    data_dir: "dogfood/data"
```

Comment:
our existing dogfood content is in `dogfood/data/source` are you intending to move this as part of the PR?

Reply:
I did move it, so it aligned with the defaults ... I'm sure I did 😓